### PR TITLE
Adding "_blank" target to post links

### LINKED ISSUE #20: Post links need to open in New tab
Need to figure out a way for converting all links in the posts to have `target` property set to `_blank`

### PR DESCRIPTION
**Pull request by priyasonali**
This PR fixes #20 

## Status: Done

### CODE REVIEW

**Comment by cod3rguy** [comment]
Let me know when you're done, so I review and pull in.

**Comment by priyasonali** [comment]
Ready for review.

File: _includes/footer.html [hunk 1]
```diff
@@ -14,5 +14,8 @@
     </script>
     <!--End of Tawk.to Script-->
     {% endif %}
+    <!-- Start of script for adding external target to links in posts -->
+    <script type="text/javascript" src="{{site.url}}/scripts/script.js"></script>
+    <!-- Start of script for adding external target to links in posts -->
```
Comment (cod3rguy) [inline _includes/footer.html]: Shouldn't it be "End of script..." ? 😜

Comment (priyasonali) [inline _includes/footer.html]: Fixed it 😝

**Pull request merged**